token-tracker/src/lib/context.js

```diff
@@ -0,0 +1,89 @@
+/** @typedef { import("./contract-helper.js").Contract } Contract */
+
+import { initWeb3 } from "./ethereum-helper.js"
+
+export default class Context {
+  /** @property {Web3} web3 */
+
+  /**
+   * @typedef {Map<String,Contract>} Contracts
+   * @type {Contracts}
+   */
+  #contracts
+
+  /**
+   * Deployment block is considered a starting
+   * block number that can be used as 'fromBlock' for ethereum events lookup.
+   * @type {Number}
+   */
+  #deploymentBlock
```

Comment:
What do you think about naming `deploymentBlock` to something more specific, ex. `deploymentKeepTokenBlock`? If we add `lp tokens` to `truth-sources` then we also need to take into account deployment blocks of `lp tokens` and probably add them in here as well.

Reply:
Actually, I take it back, sorry. KEEP is the "main" token here and LP tokens will be in the truth sources. I'll have deployments block of LP tokens defined in LP specific code.